edgelet/Cargo.lock

```diff
@@ -2009,43 +2000,20 @@ checksum = "41cc0f7e4d5d4544e8861606a285bb08d3e70712ccc7d2b84d7c0ccfaf4b05ce"
 
 [[package]]
 name = "regex"
-version = "0.2.11"
+version = "1.4.6"
```

Comment:
Since you've updated the 1.x one's version as well (before your change it was 1.3.9), might as well update it to latest 1.5.4 ?

Reply:
Cargo is unifying this regex with IIS's regex version. I am working on a PR for IIS also.

Reply:
```
$ cargo update -p regex:1.3.9 --precise 1.5.4

    Updating crates.io index
    Removing aho-corasick v0.6.10
    Removing aho-corasick v0.7.13
      Adding aho-corasick v0.7.18
    Updating memchr v2.3.3 -> v2.4.1
    Removing regex v0.2.11
    Removing regex v1.3.9
      Adding regex v1.5.4
    Removing regex-syntax v0.5.6
    Removing regex-syntax v0.6.18
      Adding regex-syntax v0.6.25
    Removing thread_local v0.3.6
    Removing thread_local v1.0.1
    Removing ucd-util v0.1.8
    Removing utf8-ranges v1.0.4

$ grep -A1 '= "regex"' Cargo.lock 

name = "regex"
version = "1.5.4"
```

Reply:
Huh, alright. Will do.